pages/common/tar.md

```diff
@@ -31,3 +31,7 @@
 - Extract files matching a pattern:
 
 `tar xf {{source.tar}} --wildcards {{"*.html"}}`
+
+- Extract a file or folder the folder structure in the archive:
```

Comment:
The description is incorrect. There have been many suggested alternatives in the review. Let's go with one of them.

Reply:
Agreed, I thought I put that in there, but apparently not. Updated now.